### DISCUSSION TITLE
Make MonitorControl start work before starting MacOS (before login)?

### DISCUSSION THREAD

**Discussion by slepoy1** [MonitorControl/MonitorControl#1153] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues

### Discussion

Hello. Is it possible to make MonitorControl start work before typing password (during OS loading)? Thanks.

**Comment by waydabber** [comment]
Unfortunately this is not normally possible. You might be able to use `launchctl` to start the app as root but this is entirely untested and it might not produce the desired results (also the app quite heavily relies on the menubar being available).
